src/lib/sw_engine/tvgSwRle.cpp

```diff
@@ -613,6 +613,94 @@ static int _genRle(RleWorker& rw)
 }
 
 
+void _intersectSpansRegion(const SwRleData *clip, SwSpan *spans, SwSpan *end, SwSpan **outSpans, uint32_t available)
+{
+    SwSpan *out = *outSpans;
+    int16_t sx1, sx2, cx1, cx2, x;
+    uint16_t len;
+    const SwSpan *clipSpans = clip->spans;
+    const SwSpan *clipEnd = clip->spans + clip->size;
+
+    while (available && spans < end ) {
+        if (clipSpans > clipEnd) {
+            spans = end;
+            break;
+        }
+        if (clipSpans->y > spans->y) {
+            ++spans;
+            continue;
+        }
+        if (spans->y != clipSpans->y) {
+            ++clipSpans;
+            continue;
+        }
+        sx1 = spans->x;
```

Comment:
Declare when it's begun to use
auto sx1 = ...
auto sx2 = 
auto cx1 = 
auto cx2 =

Reply:
changed it